scripts/install-sdk.sh

```diff
@@ -50,9 +50,18 @@ FORCE=
 
 updatePackage() {
     name=$1
+    packageData=`"$NODE" -e 'var package = require("./package.json");console.log(package.c9plugins["'$name'"] ? package.c9plugins["'$name'"].substr(1) : package.devPlugins["'$name'"] ? package.devPlugins["'$name'"].substr(1) : "origin/master")'`;
```

Comment:
Did you keep `package.devPlugins` to not have merge conflicts in package.json?
It looks like value from devPlugins won't be used in any case.